### PR TITLE
Feature/Quarkus scripts output clean-up

### LINKED ISSUE #131: Quarkus scripts output clean-up
Quarkus scripts output clean-up: Add "-B" flag to all the maven commands in the scripts to delete unnecessary logs.

### PR DESCRIPTION
**Pull request by BenjaminE98**
Closes #131 

simple output-log cleanup (suppresses unnecessary logs)

### CODE REVIEW

File: scripts/pipelines/azure-devops/templates/test/quarkus-test.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-mvn test -Dmaven.install.skip=true -Pnative
+mvn test -B -Dmaven.install.skip=true -Pnative
```
Comment (shiftwolf) [inline scripts/pipelines/azure-devops/templates/test/quarkus-test.sh:2]: Is there a reason to not set this to --quiet unlike the others?

File: scripts/pipelines/azure-devops/templates/build/quarkus-build.sh [hunk 2]
```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-mvn package -Pnative
\ No newline at end of file
+mvn package --quiet -B -Pnative
```
Comment (albertdb) [inline scripts/pipelines/azure-devops/templates/build/quarkus-build.sh:2]:
```suggestion
mvn package -B -Pnative
```

File: scripts/pipelines/azure-devops/templates/build/quarkus-jvm-build.sh [hunk 3]
```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-mvn package
\ No newline at end of file
+mvn package --quiet -B
```
Comment (albertdb) [inline scripts/pipelines/azure-devops/templates/build/quarkus-jvm-build.sh:2]:
```suggestion
mvn package -B
```

File: scripts/pipelines/azure-devops/templates/library-package/java-library-package.sh [hunk 4]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 settings_path="$(pwd)/.pipelines/scripts/settings.xml"
-mvn deploy -Dmaven.install.skip=true -Dmaven.test.skip=true --settings $settings_path
\ No newline at end of file
+mvn deploy --quiet -B -Dmaven.install.skip=true -Dmaven.test.skip=true --settings $settings_path
```
Comment (albertdb) [inline scripts/pipelines/azure-devops/templates/library-package/java-library-package.sh:3]:
```suggestion
mvn deploy -B -Dmaven.install.skip=true -Dmaven.test.skip=true --settings $settings_path
```

File: scripts/pipelines/azure-devops/templates/test/quarkus-jvm-test.sh [hunk 5]
```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-mvn test -Dmaven.install.skip=true
+mvn test --quiet -B -Dmaven.install.skip=true
```
Comment (albertdb) [inline scripts/pipelines/azure-devops/templates/test/quarkus-jvm-test.sh:2]:
```suggestion
mvn test -B -Dmaven.install.skip=true
```

**Review by albertdb — CHANGES_REQUESTED** [review]
`--quiet` is too quiet.

**Review by albertdb — APPROVED** [review]

**Pull request merged**